test/functions.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+require '../functions.php';
+
+use PHPUnit\Framework\TestCase;
+
+class functions extends TestCase
+{
+    public function testformatRecordsForDisplaySuccess()
+    {
+        $input = ["Artist"=>"artist","Title"=> "title", "Rating"=>"rating", "Released"=>1993, "Condition"=>"condition"];
```

Comment:
you can use single quotes - I don't know if it's something that the guys would flag <shrug>

also make sure that there are spaces before and after =>